### PR TITLE
Change roster so roster displays all students sorted

### PR DESCRIPTION
**Pull request by rwobben**
Here the second step. 

Roster now has no parameters and displays all students in a course. 

solves #271

### CODE REVIEW

**Comment by rwobben** [comment]
Right. that was the last step I wanted to do

**Comment by rwobben** [comment]
Oke, 

Found a answer : 

Could you live with this code as a example code : 

     public IEnumerable<string> Roster()
    {
        var students = new List<string>(); 

        foreach (var item in roster)
        {
            foreach (var student in item.Value)
            {
                students.Add(student); 
                    
            }
        }

        return students;

    }

**Comment by ErikSchierboom** [comment]
Sure, but you forgot to add the ordering :)

**Comment by rwobben** [comment]
I did not  forget it. 
With this code all tests are already sorted. 

This test is green where the elements are not added in sorted order 

     public IEnumerable<string> Roster()
    {
        var students = new List<string>(); 

        foreach (var item in roster)
        {
            foreach (var student in item.Value)
            {
                students.Add(student); 
                    
            }
        }

        return students;

    }

**Comment by ErikSchierboom** [comment]
@rwobben Ah yes, you're right! If you could update the PR with the modified Example.cs, the build should succeed again.

File: exercises/grade-school/Example.cs [hunk 1]
```diff
@@ -13,7 +13,21 @@ public void Add(string student, int grade)
             roster.Add(grade, new SortedList<string> { student });
     }
 
-    public IEnumerable<string> Roster(int grade) => roster[grade];
+   public IEnumerable<string> Roster()
+   {
+    var students = new List<string>();
```
Comment (ErikSchierboom) [inline exercises/grade-school/Example.cs]: The indentation of all lines in the `Roster` method are off. They should all be indented one tab more.

**Comment by ErikSchierboom** [comment]
Okay, looking good! All that's left is to wait for the CI builds to pass.

**Comment by ErikSchierboom** [comment]
Merged! Thanks! :tada:

**Pull request merged**